content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -63,6 +63,7 @@ Kubernetes starts with four initial namespaces:
    * `kube-system` The namespace for objects created by the Kubernetes system
    * `kube-public` This namespace is created automatically and is readable by all users (including those not authenticated). This namespace is mostly reserved for cluster usage, in case that some resources should be visible and readable publicly throughout the whole cluster. The public aspect of this namespace is only a convention, not a requirement.
    * `kube-node-lease` This namespace for the lease objects associated with each node which improves the performance of the node heartbeats as the cluster scales.
+   * `kube-apiserver-lease` TODO(roycaihw): behind feature gate
```

Comment:
Remember to change “four” to “several”. Also, I suggest sorting the list.

Here are some wording suggestions that can either go in this PR _or_ land after this one merges.

```suggestion
   * `kube-apiserver-lease` *⚠????TODO????TODO????⚠* _behind feature gate_
   * `kube-node-lease` A namespace for the lease objects associated with each node. Node leases improve the performance of node heartbeats for large clusters.
   * `kube-public` A namespace that is created automatically and is readable by all users (including those not authenticated). This namespace is mostly reserved for cluster usage, in case some resources should be visible and readable publicly throughout the whole cluster. 
   * `kube-system` General namespace for Kubernetes system objects.

{{< note >}}
Making the `kube-public` namespace and its contents world-readable is a convention, not a requirement.
{{< /note >}}
```

Reply:
Thanks for the suggestion. We decided to reuse `kube-system` for this feature. Reverted the change.